f5_openstack_agent/lbaasv2/drivers/bigip/agent_manager.py

```diff
@@ -149,7 +149,12 @@
         'esd_auto_refresh',
         default=True,
         help='Enable ESD file periodic refresh'
-    )
+    ),
+    cfg.StrOpt(
+        'f5_extended_profile_name',
+        default='',
+        help=('The file name of the http profile content of a listener')
```

Comment:
This json file is for all possible profiles and other customizable properties, not only for http_profile.

Reply:
done